### PR TITLE
Installation added to readme file

### PR DESCRIPTION
**Pull request by omergulen**
As an API v2 user, I encountered with this problem, I needed to install it but since there wasn't clear information about if it is available in pip I tried and saw that it is available, but with adding these tutorial lines it would be easier for the new users. It also includes the downloading and installing it via `setup.py` version of the installation.

Need to be added:

- [ ] Features
- [ ] Dependencies

### CODE REVIEW

**Comment by yahoocla** [comment]
Thank you for submitting this pull request, however I do not see a valid CLA on file for you. Before we can merge this request please visit https://yahoocla.herokuapp.com/ and agree to the terms. Thanks! :smile:

**Comment by omergulen** [comment]
How much does it take to answer to a PR normally?

File: README.rst [hunk 1]
```diff
@@ -1,8 +1,27 @@
...
+.. code-block:: bash
+
+    $ pip install pytumblr
+
+Install from source:
+
+.. code-block:: bash
+
+    $ git clone https://github.com/tumblr/pytumblr.git
+    $ cd 3taps
```
[the remark this answers is not in the document]
Comment (omergulen) [reply README.rst]: I just fixed it, my bad. 👍

**Comment by omergulen** [comment]
Fixed it!

**Comment by omergulen** [comment]
Waiting for review 👍

**Comment by komapa** [comment]
Thank you for the contribution!

**Pull request merged**